app/src/main/res/values/strings.xml

```diff
@@ -78,6 +78,8 @@
     <string name="use_inexact_seek_title">Use fast inexact seek</string>
     <string name="use_inexact_seek_summary">Inexact seek allows the player to seek to positions faster with reduced precision. Seeking for 5, 15 or 25 seconds doesn\'t work with this</string>
     <string name="seek_duration_title">Fast-forward/-rewind seek duration</string>
+    <string name="progressive_load_interval_title">Playback Load Interval Size</string>
```

Comment:
Not sure about the writing but I think it should look like this:
```suggestion
    <string name="progressive_load_interval_title">Playback load interval size</string>
```

Reply:
Done.